src/lib/encoding.ts

```diff
@@ -295,6 +296,18 @@ export function parseManifestStream(stream: Readable): EventEmitter {
         paths = {};
       }
     }
+    if (
```

Comment:
Please try to keep the code style of the surrounding code blocks, i.e. a blank line followed by a comment that highlights what the subsequent conditional block is intending to select on during parsing, followed by the conditional block itself.